starboard/nplb/player_create_test.cc

```diff
@@ -323,20 +323,13 @@ TEST_F(SbPlayerTest, MultiPlayer) {
       kSbPlayerOutputModeDecodeToTexture, kSbPlayerOutputModePunchOut};
 
   constexpr SbMediaAudioCodec kAudioCodecs[] = {
-    kSbMediaAudioCodecNone,
-
-    kSbMediaAudioCodecAac,
-    kSbMediaAudioCodecAc3,
-    kSbMediaAudioCodecEac3,
-    kSbMediaAudioCodecOpus,
-    kSbMediaAudioCodecVorbis,
-#if SB_API_VERSION >= 14
-    kSbMediaAudioCodecMp3,
-    kSbMediaAudioCodecFlac,
-    kSbMediaAudioCodecPcm,
-#endif  // SB_API_VERSION >= 14
+      kSbMediaAudioCodecNone,
```

Comment:
Fix the formatting?

Reply:
This is what the linter wanted. I tried changing back but I think the `#`s are messing with it somehow.

Reply:
Please use `// clang-format off` `// clang-format on` for the block, if it refuses to behave otherwise.